.github/workflows/ci.yml

```diff
@@ -60,3 +59,43 @@ jobs:
       with:
         name: 'vbamc_packages_${{ matrix.configuration }}'
         path: '${{ github.workspace }}/dist'
+
+  benchmark:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: ['ubuntu-22.04', 'windows-2022', 'macos-13.0']
+
+    env:
+      Configuration: 'Release'
+
+    steps:
+    - name: checkout
+      uses: actions/checkout@v3
+      with:
+        submodules: true
+
+    - name: setup dotnet
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 8
+
+    - name: restore
+      run: dotnet restore
+
+    - name: build
+      run: dotnet build --no-restore -c Release tests/VbaCompiler.Benchmark/VbaCompiler.Benchmark.csproj
+
+    - name: benchmark
+      shell: bash
+      run: |
+        dotnet run -e github -c Release --artifacts --project tests/VbaCompiler.Benchmark/VbaCompiler.Benchmark.csproj
```

Comment:
What are the `-e github` and `--artifacts ` used  for? Could not find anything about those options in the [official documentation](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-run).

Reply:
`dotnet run` will pass some of the arguments to the underlying tool - in this case DotnetBenchmark.

https://github.com/dotnet/BenchmarkDotNet/blob/master/docs/articles/guides/console-args.md

I should rewrite it so it is more clear which arguments are for `dotnet run` and which are for benchmarking.

Reply:
The `-e github` changes DotnetBenchmark reporter so the output works nicely in GitHub Actions.

The `--artifacts` forces DotnetBenchmark to generate the results files in markdown format. This is done by default but for some reason it must be set explicitly when used with `-e github`.